x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/tabs/helpers.test.tsx

```diff
@@ -38,11 +37,11 @@ describe('helpers', () => {
     });
 
     test('it returns false when `enrichedFieldMetadata` contains an @timestamp field', () => {
-      expect(showMissingTimestampCallout([timestamp, eventCategory, someField])).toBe(false);
+      expect(showMissingTimestampCallout([timestamp, eventCategory])).toBe(false);
     });
 
     test('it returns true when `enrichedFieldMetadata` does NOT contain an @timestamp field', () => {
-      expect(showMissingTimestampCallout([eventCategory, someField])).toBe(true);
+      expect(showMissingTimestampCallout([eventCategory])).toBe(true);
```

Comment:
For reviewers:

After narroweding down types turns out only `ecsBasedFieldMetadata` is and should be passed into `showMissingTimestampCallout`, hence `someField` being a custom field mock and yielding a type error is getting removed